roms_tools/tiling/partition.py

```diff
@@ -296,59 +297,72 @@ def cumsum(pmf):
 
 
 def partition_netcdf(
-    filepath: str | Path,
+    filepath: str | Path | Sequence[str | Path],
     np_eta: int = 1,
     np_xi: int = 1,
+    output_dir: str | Path | None = None,
     include_coarse_dims: bool = True,
 ) -> None:
```

Comment:
typehint mismatch to actual return value

Reply:
> This should have been caught by pre-commit but I'm just noticing there's no MyPy on pre-commit?

I opened https://github.com/CWorthy-ocean/roms-tools/pull/405